docs/contents/users/build.md

```diff
@@ -31,7 +31,7 @@ make docker-push
 ## Build Remaining Containers
 
 The second step of the build is to build the rest of the containers out
-of the [EKS Distro Build Steps Repository](https://github.com/aws/eks-distro).
+of the [EKS Distro Container Repository](https://github.com/aws/eks-distro).
```

Comment:
Should it just be `EKS Distro Repository`, without the Container?

Reply:
Yeh, that is what we call it and I'm fine with that, but I thought this name was a bit more descriptive

Reply:
I'm inclined for `EKS Distro Repository`

Reply:
Done